### PR TITLE
feat(etl): new etl mapping for npi-pro

### PR DESCRIPTION
**Pull request by thanh-nguyen-dang**

### CODE REVIEW

File: qa-covid19.planx-pla.net/portal/gitops.json [hunk 1]
```diff
@@ -807,6 +807,34 @@
...
             "field": "total_in_use_beds_covid",
             "name": "IL Total In Use COVID-19 Beds"
+          },
+          {
+            "field": "npi",
+            "name": "National Provider Identifier"
+          },
+          {
+            "field": "provider_business_practice_city",
+            "name": "Provider Business Practice City"
```
Comment (mfshao) [inline qa-covid19.planx-pla.net/portal/gitops.json:817]: this is redundant and can be removed, guppy will auto captilaized 1st letter in each word divided by `_`

you only need to use this section to customize fied name if above rule cannot satisfy your needs

File: qa-covid19.planx-pla.net/portal/gitops.json [hunk 2]
```diff
@@ -807,6 +807,34 @@
...
+            "field": "provider_type",
+            "name": "Provider Type"
+          },
+          {
+            "field": "provider_subtype",
+            "name": "Provider Subtype"
+          },
+          {
+            "field": "detailed_specialty",
+            "name": "Detailed Specialty"
```
Comment (mfshao) [inline qa-covid19.planx-pla.net/portal/gitops.json:837]: the same for above 3 fields

File: qa-covid19.planx-pla.net/etlMapping.yaml [hunk 3]
```diff
@@ -18,6 +18,11 @@ mappings:
       - name: reporting_org
       - name: state_hospital_region
       - name: state_region_description
+      - name: npi
+      - name: provider_first_line_business_pr
+      - name: provider_second_line_business_pr
```
Comment (paulineribeyre) [inline qa-covid19.planx-pla.net/etlMapping.yaml:23]: these 2 are not in the portal config

Comment (thanh-nguyen-dang) [reply qa-covid19.planx-pla.net/etlMapping.yaml:23]: https://docs.google.com/spreadsheets/d/1Cdpyi6hzdoKqLuvvB4bNGkJPS81pVzb-RchzajHLydE/edit#gid=235693014
it is not in the specification

Comment (paulineribeyre) [reply qa-covid19.planx-pla.net/etlMapping.yaml:23]: yeah but the spreadsheet is just a guide, there's no need to add to the ETL mapping if you don't use it 🤷

Comment (thanh-nguyen-dang) [reply qa-covid19.planx-pla.net/etlMapping.yaml:23]: for querying maybe? anyway, what are the titles to display?

Comment (thanh-nguyen-dang) [reply qa-covid19.planx-pla.net/etlMapping.yaml:23]: I found many fields in etlMapping but not in portal config anyway @paulineribeyre

Comment (paulineribeyre) [reply qa-covid19.planx-pla.net/etlMapping.yaml:23]: ideally only the fields are we show in the portal (as described in the spreadsheet) should be in the ETL mapping

Comment (paulineribeyre) [reply qa-covid19.planx-pla.net/etlMapping.yaml:23]: no one uses the query

**Pull request merged**